fhir-server/src/main/java/au/csiro/pathling/aggregate/AggregateRequest.java

```diff
@@ -97,105 +105,13 @@ private static ResourceType getSubjectResource(@Nonnull final Parameters paramet
   }
 
   @Nonnull
-  private static List<Aggregation> getAggregations(@Nonnull final Parameters parameters) {
-    return parameters.getParameter().stream()
-        .filter(param -> param.getName().equals("aggregation"))
-        .map(aggregation -> {
-          final Optional<String> label = aggregation.getPart()
-              .stream()
-              .filter(
-                  part -> part.getName().equals("label") && part.getValue() instanceof StringType)
-              .findFirst()
-              .map(parameter -> parameter.getValue().toString());
-          final String expression = aggregation.getPart()
-              .stream()
-              .filter(part -> part.getName().equals("expression") &&
-                  part.getValue() instanceof StringType)
-              .findFirst()
-              .map(parameter -> parameter.getValue().toString())
-              .orElseThrow(() -> new InvalidUserInputError("Aggregation must have expression"));
-          return new Aggregation(label, expression);
-        })
-        .collect(Collectors.toList());
-  }
-
-  @Nonnull
-  private static List<Grouping> getGroupings(@Nonnull final Parameters parameters) {
+  private static List<String> getExpressions(@Nonnull final Parameters parameters,
+      @Nonnull final String parameter) {
     return parameters.getParameter().stream()
-        .filter(param -> param.getName().equals("grouping"))
-        .map(grouping -> {
-          final Optional<String> label = grouping.getPart()
-              .stream()
-              .filter(
-                  part -> part.getName().equals("label") && part.getValue() instanceof StringType)
-              .findFirst()
-              .map(parameter -> parameter.getValue().toString());
-          final String expression = grouping.getPart()
-              .stream()
-              .filter(part -> part.getName().equals("expression") &&
-                  part.getValue() instanceof StringType)
-              .findFirst()
-              .map(parameter -> parameter.getValue().toString())
-              .orElseThrow(() -> new InvalidUserInputError("Grouping must have expression"));
-          return new Grouping(label, expression);
-        })
+        .filter(
+            param -> param.getName().equals(parameter) && param.getValue() instanceof StringType)
+        .map(aggregation -> aggregation.getValue().toString())
```

Comment:
change 'aggregation' to 'expression'?

Reply:
Fixed in 5a7d60235f3e49e57a0b5483db8b164e88e71187.